docs/components/zeebe/deployment-guide/interceptors.md

```diff
@@ -0,0 +1,172 @@
+---
+id: interceptors
+title: "Interceptors"
+sidebar_label: "Interceptors"
+---
+
+> This functionality is currently only available in Camunda Cloud Self-Managed.
+
+All communication from a client to a broker must first pass through a gateway.
+There they can be intercepted before being dispatched. Zeebe provides a way to
+load arbitrary interceptors into the gateway. Some typical examples of what you
+can accomplish with this include:
+
+- enforcing custom authorization rules on incoming calls
+- monitoring and logging of incoming calls
+
+## Implementing an interceptor
+For the communication between client and gateway, Zeebe uses the gRPC
+[protocol](/components/zeebe/technical-concepts/protocols.md). An interceptor is
+thus implemented as a gRPC
+[ServerInterceptor](https://grpc.github.io/grpc-java/javadoc/io/grpc/ServerInterceptor.html).
+
+An implementation must adhere to the following requirements:
+- it implements
+  [ServerInterceptor](https://grpc.github.io/grpc-java/javadoc/io/grpc/ServerInterceptor.html)
+- it has public visibility
+- it has a public default constructor (i.e. no-arg constructor)
+
+Let's consider an interceptor that provides logging of incoming calls as an
+example. Other ServerInterceptor examples can be found in the official grpc-java
+[examples](https://github.com/grpc/grpc-java/tree/v1.41.0/examples).
```

Comment:
:question: Should we point to master instead of a specific tagged version? :thinking: